app/src/app/[lng]/[inventory]/preferences/activities/PreferenceCard.tsx

```diff
@@ -0,0 +1,50 @@
+import type { TFunction } from "i18next";
+import { Button, Text, VStack } from "@chakra-ui/react";
+import Image from "next/image";
+
+export function PreferenceCard({
+  onClick,
+  src,
+  title,
+  t,
+}: {
+  onClick: () => void;
```

Comment:
Might be good to have this as optional and provide an empty function as the default value.

Reply:
We are going to need it when we build the backend and have to call the API